src/main.js

```diff
@@ -76,7 +76,7 @@ require.config({
 
 var templateEngine;
 require([
-  'jquery', 'TemplateEngine'
+  'jquery', 'lib/TemplateEngine'
```

Comment:
This can work (I have to check the code), but usually the paths go into  the paths section of the configs and here only the name is used. As it's already there (see line 30 of this file) I guess I need to dig deeper here